tests/test_math.scad

```diff
@@ -1,586 +1,650 @@
-include <BOSL2/std.scad>
+include <BOSL/constants.scad>
```

Comment:
This is a BOSL1 include, not a BOSL2 include.

Reply:
I copied the wrong file into my fork.  I have updated the file to the right version.